plugins/chat/config/locales/client.en.yml

```diff
@@ -693,8 +693,8 @@ en:
       titles:
         chat_mention: "Chat mention"
         chat_invitation: "Chat invitation"
```

Comment:
perhaps this should be "Invited to chat channel"

Reply:
I could go either way - changed it to your suggestion.